src/c++/examples/CMakeLists.txt

```diff
@@ -240,6 +240,20 @@ if(TRITON_ENABLE_CC_GRPC)
     RUNTIME DESTINATION bin
   )
 
+  #
+  # simple_grpc_trace_client
+  #
+  add_executable(simple_grpc_trace_client simple_grpc_trace_client.cc)
+  target_link_libraries(
+    simple_grpc_trace_client
+    PRIVATE
+      grpcclient_static
+  )
+  install(
+    TARGETS simple_grpc_trace_client
+    RUNTIME DESTINATION bin
+  )
+
```

Comment:
I am inclined to add new unit test suit instead of adding new dedicated client for new feature (see [discussion](https://github.com/triton-inference-server/client/pull/71#issuecomment-1057301700)), you can add new test suit (with proper nameing) in [cc_client_test.cc](https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/tests/cc_client_test.cc), and it should be able to be picked up in the test. i.e. in [L0_grpc](https://github.com/triton-inference-server/server/blob/main/qa/L0_grpc/test.sh#L422), the test with "GRPC" prefix will be run

Reply:
Thanks for the review. Will add new test suit for this.

Reply:
Updated, please review.